### ISSUE TITLE
[Drawing] - Colors not working

### ISSUE THREAD

**Issue by 132ndNeck** [birgersp/dcs-bajas#45]
When trying to draw in various colors, it does not seem to work.
I cant select other colors (I guess its because it has not been activated/programmed yet,, but just wanted to give you a heads up

**Comment by jkhoel** [comment]
Yes, the whole drawing thing has dragged out a bit. I've had to go back and change a lot of the code to make everything work and look as I wanted to. Currently working on inputting coordinates correctly when editing the drawn features. Colors are fairly simple, so saving them for last :)
